### PR TITLE
updated default parameters

### PR DESCRIPTION
**Pull request by billingsley-john**
PR to update the default parameters for all parametric_components. The current material_tag setter in Develop raises a ValueError for some default parameters. This is being fixed in #105 so this branch should be merged after #105 has been merged

### CODE REVIEW

**Comment by Shimwell** [comment]
I might have a small change to make to this before it is merged

File: paramak/parametric_components/blanket_constant_thickness_arc_h.py [hunk 1]
```diff
@@ -51,13 +51,13 @@ def __init__(
...
         points=None,
-        stp_filename=None,
+        stp_filename="BlanketConstantThicknessArcH.stp",
         rotation_angle=360,
         azimuth_placement_angle=0,
         solid=None,
         color=None,
         name=None,
-        material_tag=None,
+        material_tag="blanket_material",
```
Comment (Shimwell) [inline paramak/parametric_components/blanket_constant_thickness_arc_h.py:60]:
```suggestion
        material_tag="blanket_mat",
```

File: paramak/parametric_components/blanket_constant_thickness_fp.py [hunk 2]
```diff
@@ -73,13 +73,13 @@ def __init__(
...
         points=None,
-        stp_filename=None,
+        stp_filename="BlanketConstantThicknessFP.stp",
         rotation_angle=360,
         azimuth_placement_angle=0,
         solid=None,
         color=None,
         name=None,
-        material_tag=None,
+        material_tag="blanket_material",
```
Comment (Shimwell) [inline paramak/parametric_components/blanket_constant_thickness_fp.py:82]:
```suggestion
        material_tag="blanket_mat",
```

File: paramak/parametric_components/center_column_circular.py [hunk 3]
```diff
@@ -28,11 +28,11 @@ def __init__(
         workplane="XZ",
         rotation_angle=360,
         solid=None,
-        stp_filename="center_column.stp",
+        stp_filename="CenterColumnShieldCircular.stp",
         color=None,
         points=None,
         name="center_column",
-        material_tag="center_column_material",
+        material_tag="center_column_shield_material",
```
Comment (Shimwell) [inline paramak/parametric_components/center_column_circular.py:35]:
```suggestion
        material_tag="center_column_shield_mat",
```

This one is the reason for changing material to mat as it is too long for the DAGMC limit

File: paramak/parametric_components/center_column_cylinder.py [hunk 4]
```diff
@@ -24,10 +24,10 @@ def __init__(
         name=None,
         color=None,
         workplane="XZ",
-        stp_filename="center_column_shield.stp",
+        stp_filename="CenterColumnShieldCylinder.stp",
         rotation_angle=360,
         solid=None,
-        material_tag=None,
+        material_tag="center_column_shield_material",
```
Comment (Shimwell) [inline paramak/parametric_components/center_column_cylinder.py:30]:
```suggestion
        material_tag="center_column_shield_mat",
```

File: paramak/parametric_components/center_column_flat_top_circular.py [hunk 5]
```diff
@@ -16,11 +16,11 @@ def __init__(
         workplane="XZ",
         rotation_angle=360,
         solid=None,
-        stp_filename="center_column.stp",
+        stp_filename="CenterColumnShieldFlatTopCircular.stp",
         color=None,
         points=None,
         name="center_column",
-        material_tag="center_column_material",
+        material_tag="center_column_shield_material",
```
Comment (Shimwell) [inline paramak/parametric_components/center_column_flat_top_circular.py:23]:
```suggestion
        material_tag="center_column_shield_mat",
```

File: paramak/parametric_components/center_column_flat_top_hyperbola.py [hunk 6]
```diff
@@ -30,11 +30,11 @@ def __init__(
         workplane="XZ",
         rotation_angle=360,
         solid=None,
-        stp_filename="center_column.stp",
+        stp_filename="CenterColumnFlatTopHyperbola.stp",
         color=None,
         points=None,
         name="center_column",
-        material_tag="center_column_material",
+        material_tag="center_column_shield_material",
```
Comment (Shimwell) [inline paramak/parametric_components/center_column_flat_top_hyperbola.py:37]:
```suggestion
        material_tag="center_column_shield_mat",
```

File: paramak/parametric_components/center_column_plasma_dependant.py [hunk 7]
```diff
@@ -35,8 +35,8 @@ def __init__(
         points=None,
         name=None,
         color=None,
-        material_tag=None,
-        stp_filename=None,
+        material_tag="center_column_shield_material",
```
Comment (Shimwell) [inline paramak/parametric_components/center_column_plasma_dependant.py:38]:
```suggestion
        material_tag="center_column_shield_mat",
```

File: paramak/parametric_components/inner_tf_coils_circular.py [hunk 8]
```diff
@@ -29,12 +29,12 @@ def __init__(
...
         distance=None,
         workplane="XY",
-        stp_filename=None,
+        stp_filename="InnerTfCoilsCircular.stp",
         solid=None,
         color=None,
         azimuth_placement_angle=0,
         cut=None,
-        material_tag=None,
+        material_tag="tf_coil_material",
```
Comment (Shimwell) [inline paramak/parametric_components/inner_tf_coils_circular.py:37]:
```suggestion
        material_tag="inner_tf_coil_mat",
```

File: paramak/parametric_components/inner_tf_coils_flat.py [hunk 9]
```diff
@@ -29,12 +29,12 @@ def __init__(
...
         distance=None,
         workplane="XY",
-        stp_filename=None,
+        stp_filename="InnerTfCoilsFlat.stp",
         solid=None,
         color=None,
         azimuth_placement_angle=0,
         cut=None,
-        material_tag=None,
+        material_tag="tf_coil_material",
```
Comment (Shimwell) [inline paramak/parametric_components/inner_tf_coils_flat.py:37]:
```suggestion
        material_tag="inner_tf_coil_mat",
```

File: paramak/parametric_components/divertor_block.py [hunk 10]
```diff
@@ -47,10 +47,10 @@ def __init__(
         name=None,
         rotation_angle=360,
         solid=None,
-        stp_filename="divertor.stp",
+        stp_filename="DivertorBlock.stp",
         color=None,
         azimuth_placement_angle=0,
-        material_tag=None,
+        material_tag="divertor_material",
```
Comment (Shimwell) [inline paramak/parametric_components/divertor_block.py:53]:
```suggestion
        material_tag="divertor_mat",
```

File: paramak/parametric_components/poloidal_field_coil.py [hunk 11]
```diff
@@ -23,12 +23,12 @@ def __init__(
...
         rotation_angle=360,
         solid=None,
-        stp_filename="poloidal_field_coil.stp",
+        stp_filename="PoloidalFieldCoil.stp",
         color=None,
         azimuth_placement_angle=0,
         points=None,
         name=None,
-        material_tag=None,
+        material_tag="poloidal_field_coil_material",
```
Comment (Shimwell) [inline paramak/parametric_components/poloidal_field_coil.py:31]:
```suggestion
        material_tag="poloidal_field_coil_mat",
```

File: paramak/parametric_components/poloidal_field_coil_case.py [hunk 12]
```diff
@@ -27,12 +27,12 @@ def __init__(
...
         rotation_angle=360,
         solid=None,
-        stp_filename="poloidal_field_coil_case.stp",
+        stp_filename="PoloidalFieldCoilCase.stp",
         color=None,
         azimuth_placement_angle=0,
         points=None,
         name=None,
-        material_tag=None,
+        material_tag="poloidal_field_coil_case_material",
```
Comment (Shimwell) [inline paramak/parametric_components/poloidal_field_coil_case.py:35]:
```suggestion
        material_tag="poloidal_field_coil_case_mat",
```

Comment (Shimwell) [inline paramak/parametric_components/poloidal_field_coil_case.py:35]:
```suggestion
        material_tag="pf_coil_case_material",
```

Comment (Shimwell) [inline paramak/parametric_components/poloidal_field_coil_case.py:35]:
```suggestion
        material_tag="pf_coil_case_matl",
```

Comment (Shimwell) [inline paramak/parametric_components/poloidal_field_coil_case.py:35]:
```suggestion
        material_tag="pf_coil_case_mat",
```

**Comment by billingsley-john** [comment]
I don't think all of my commits pushed correctly! Thanks for the changes @Shimwell

File: paramak/parametric_components/poloidal_field_coil.py (hunk 11 above)
Comment (Shimwell) [inline paramak/parametric_components/poloidal_field_coil.py:31]:
```suggestion
        material_tag="pf_coil_mat",
```

File: paramak/parametric_components/poloidal_field_coil_case_fc.py [hunk 13]
```diff
@@ -21,12 +21,12 @@ def __init__(
...
         rotation_angle=360,
         solid=None,
-        stp_filename="poloidal_field_coil_case.stp",
+        stp_filename="PoloidalFieldCoilCaseFC.stp",
         color=None,
         azimuth_placement_angle=0,
         points=None,
         name=None,
-        material_tag=None,
+        material_tag="poloidal_field_coil_case_material",
```
Comment (Shimwell) [inline paramak/parametric_components/poloidal_field_coil_case_fc.py:29]:
```suggestion
        material_tag="pf_coil_case_mat",
```

File: paramak/parametric_components/toroidal_field_coil_coat_hanger.py [hunk 14]
```diff
@@ -42,12 +42,12 @@ def __init__(
...
         rotation_angle=360,
         solid=None,
-        stp_filename="toroidal_field_coil.stp",
+        stp_filename="ToroidalFieldCoilCoatHangar.stp",
         color=None,
         azimuth_placement_angle=0,
         points=None,
         name=None,
-        material_tag=None,
+        material_tag="toroidal_field_coil_material",
```
Comment (Shimwell) [inline paramak/parametric_components/toroidal_field_coil_coat_hanger.py:50]:
```suggestion
        material_tag="outer_tf_coil_mat",
```

File: paramak/parametric_components/toroidal_field_coil_rectangle.py [hunk 15]
```diff
@@ -37,12 +37,12 @@ def __init__(
...
         rotation_angle=360,
         solid=None,
-        stp_filename="toroidal_field_coil.stp",
+        stp_filename="ToroidalFieldCoilRectangle.stp",
         color=None,
         azimuth_placement_angle=0,
         points=None,
         name=None,
-        material_tag=None,
+        material_tag="toroidal_field_coil_material",
```
Comment (Shimwell) [inline paramak/parametric_components/toroidal_field_coil_rectangle.py:45]:
```suggestion
        material_tag="outer_tf_coil_mat",
```

**Comment by Shimwell** [comment]
Bit of a shame but because we can't have "material" in some of them perhaps it is best to change to "mat" everywhere to remain consistent"

**Pull request merged**